students/nicole/lab/drills/functions.js

```diff
@@ -68,11 +68,27 @@
 // ### Infer how many arguments you'll need to pass in 
 
 // Create a function that returns the sum of three numbers
+function NumSum(numOne, numTwo, numThree){
+  return numOne + numTwo + numThree  
+};
+NumSum(4, 4, 4);
```

Comment:
Am I creating a new variable when I invoke the function?

Reply:
Great question! No, you're not creating a variable when you invoke the function.

However, if you do this:
`const sum = NumSum(4, 4, 4)`

then `sum` will be the value of the `numOne + numTwo + numThree` operation that's performed in the NumSum function. This is the power of the `return` keyword. 

You can perform a bunch of computations or do a bunch of stuff inside of a function, and once all the computing is done, store or return that final value into a variable to be used by the rest of your code.